WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -1384,6 +1392,16 @@ private extension EditableOrderViewModel {
         }
     }
 
+    /// Tax Line view models
+    /// - Parameter taxLines: order's coupon lines
+    /// - Returns: View models for the tax lines
+    ///
+    func taxLineViewModels(from taxLines: [OrderTaxLine]) -> [TaxLineViewModel] {
+        taxLines.map { TaxLineViewModel(title: "\($0.label) • \($0.rateCode) (\($0.ratePercent)%)",
```

Comment:
This string stands out as not very localizable... especially for RTL languages. Perhaps an HStack would be appropriate here, or putting the number in a number formatter so the percentage symbol ends up in the right place.

Reply:
Good point! I localized the percent here bd339575659fb1f32e9dfc195988b9de05760b59